Orange/widgets/data/owselectcolumns.py

```diff
@@ -316,6 +359,46 @@ def update_domain_role_hints(self):
         hints.update(hints_from_model("meta", self.meta_attrs))
         self.domain_role_hints = hints
 
+    @Inputs.features
+    def set_features(self, features):
+        self.features = features
+
+    def handleNewSignals(self):
+        self.check_data()
+        self.enable_used_attrs()
+        self.enable_use_features_box()
+        if self.use_input_features and len(self.features_from_data_attributes):
+            self.use_features()
+        self.unconditional_commit()
+
+    def check_data(self):
+        self.Warning.mismatching_domain.clear()
+        if self.data is not None and self.features is not None and \
+                not len(self.features_from_data_attributes):
+            self.Warning.mismatching_domain()
+
+    def enable_used_attrs(self, enable=True):
+        self.up_attr_button.setEnabled(enable)
+        self.move_attr_button.setEnabled(enable)
+        self.down_attr_button.setEnabled(enable)
+        self.used_attrs_view.setEnabled(enable)
+        self.used_attrs_view.repaint()
```

Comment:
Are you sure that these repaints (here and in the next function) are needed? It seems to work without for me.